filter/contract.go

```diff
@@ -8,35 +9,55 @@ package filter
 
 // AuthHeaders represents the common set of headers required for making
 // authorized requests
-type AuthHeaders struct{
+type AuthHeaders struct {
 	UserAuthToken    string
 	ServiceAuthToken string
 	CollectionID     string
 }
 
 // getFilterInput holds the required fields for making the GET /filters
 // API call
-type GetFilterInput struct{
+type GetFilterInput struct {
 	FilterID string
 	AuthHeaders
 }
 
 // getFilterResponse is the response body for GET /filters
-type GetFilterResponse struct{
-	ID             string              `json:"id"`
-	FilterID       string              `json:"filter_id"`
-	InstanceID     string              `json:"instance_id"`
-	Links          FilterLinks         `json:"links"`
-	Dataset        Dataset             `json:"dataset,omitempty"`
-	State          string              `json:"state"`
-	Published      bool                `json:"published"`
-	PopulationType string              `json:"population_type,omitempty"`
-	ETag           string              `json:"-"`
+type GetFilterResponse struct {
+	ID             string      `json:"id"`
+	FilterID       string      `json:"filter_id"`
+	InstanceID     string      `json:"instance_id"`
+	Links          FilterLinks `json:"links"`
+	Dataset        Dataset     `json:"dataset,omitempty"`
+	State          string      `json:"state"`
+	Published      bool        `json:"published"`
+	PopulationType string      `json:"population_type,omitempty"`
+	Custom         *bool       `json:"custom,omitempty"`
+	ETag           string      `json:"-"`
 }
 
 // FilterLinks represents the links object for /filters related endpoints
-type FilterLinks struct{
+type FilterLinks struct {
 	Dimensions Link `json:"dimensions,omitempty"`
 	Self       Link `json:"self,omitempty"`
 	Version    Link `json:"version,omitempty"`
 }
+
```

Comment:
It would be good to add doc for the CreateFlexBlueprintRequest and createFlexBlueprintResponse structs